test/evaluation/evaluationClient.spec.ts

```diff
@@ -28,11 +35,11 @@ describe('evaluate', () => {
 
   it('should return error if uri in inputDescriptors doesn\'t match', function () {
     const presentationDefinition: PresentationDefinition = getFile('./test/dif_pe_examples/pd/pd-simple-schema-age-predicate.json').presentation_definition;
```

Comment:
we had to move these to TypeScript files from json files.

ps. all json example should be moved to ts.